python/cugraph/structure/symmetrize.py

```diff
@@ -129,7 +133,8 @@ def symmetrize_ddf(df, src_name, dst_name, weight_name=None):
     return result
 
 
-def symmetrize(source_col, dest_col, value_col=None):
+def symmetrize(source_col, dest_col, value_col=None, multi=False,
+               symmetrize=True):
```

Comment:
Same with this one - it should have its docstring updated with the new args.

Reply:
Added doc